Source/Mosa.DeviceDriver/ISA/ACPI.cs

```diff
@@ -372,7 +372,8 @@ private Pointer FindBySignature(string signature)
 
 			for (var i = 0; i < (xsdt ? 16 : 8); i++)
 			{
-				var h = (ACPISDTHeader*)(xsdt ? XSDT->PointerToOtherSDT[i] : RSDT->PointerToOtherSDT[i]);
+				//https://github.com/msareedjr/MOSA-MikeOS/commit/6867064fedae707280083ba4d9ff12d468a6dce0
```

Comment:
remove comment as it doesn't provide any more useful information, descriptive comment would be better